mxcubeweb/routes/ra.py

```diff
@@ -43,25 +43,27 @@ def handle_timeout_gives_control(sid, timeout=30):
                 return make_response(msg, 409)
 
         current_user.requests_control = True
-        server.user_datastore.commit()
+        app.usermanager.update_user(current_user)
```

Comment:
`server.user_datastore.commit()` was inconsistent with other user updates in this file.